packages/importer/src/providers/standardnotes/index.ts

```diff
@@ -0,0 +1,248 @@
+      if (!data.items) {
+        errors.push(new TransformError("Backup file is invalid", file));
+        return Promise.resolve(true);
+      }
+
+      if (data.version !== SNBackupVersion) {
+        errors.push(
+          new TransformError("Backup version is not supported.", file)
+        );
+        return Promise.resolve(true);
+      }
+
+      const components: SNBackupItem[] = [];
+      const tags: SNBackupItem[] = [];
+      const snNotes: SNBackupItem[] = [];
+
+      data.items?.forEach((item) => {
+        let contentType = item.content_type;
+        switch (contentType) {
+          case ContentTypes.Note:
+            snNotes.push(item);
+          case ContentTypes.Component:
+            components.push(item);
+            break;
+          case ContentTypes.SmartTag:
+          case ContentTypes.Tag:
+            tags.push(item);
+        }
+      });
+
+      for (let item of snNotes) {
+        let type = this.getContentType(item, components);
+        item.content.editorType = type;
+        let note: Note = {
+          title: item.content.title,
+          dateCreated: item.created_at_timestamp,
+          dateEdited: item.updated_at_timestamp || item.created_at_timestamp,
+          pinned: item.content.appData["org.standardnotes.sn"]?.pinned,
+          tags: this.getTags(item, tags),
+          content: this.parseContent(item),
+        };
+        notes.push(note);
+      }
+
+      return Promise.resolve(true);
+    });
+  }
+
+  getContentType(item: SNBackupItem, components: SNBackupItem[]): EditorType {
+    let componentData =
+      item.content.appData["org.standardnotes.sn.components"] || {};
+    let editorId = Object.keys(componentData).pop();
+    if (editorId) {
+      let editor = components.find((component) => component.uuid === editorId);
+      if (editor) {
+        let contentType = {
+          ...editors[editor.content.name],
+        };
+        if (contentType.type === "code") {
+          //@ts-ignore
+          contentType.mode = componentData[editorId].mode;
+        }
+
+        return contentType;
+      }
+    }
+    return {
+      type:
+        item.content.text.includes("<") && item.content.text.includes("</")
+          ? "html"
+          : "text",
+    };
+  }
+
+  getTags(item: SNBackupItem, tags: SNBackupItem[]): string[] {
+    if (!item.content.references || item.content.references.length === 0)
+      return [];
+    let references = item.content.references;
+
+    let noteTags = [];
+    for (let reference of references) {
+      let tag = tags.find((tag) => tag.uuid === reference.uuid);
+      if (tag && tag.content.name && tag.content.name.trim() !== "") {
+        noteTags.push(tag.content.name);
+      }
+    }
+    return noteTags;
+  }
+
+  buildTableWithRows(rows: string) {
+    return `<div class="table-container">
+    <table>
+      <tbody>
+        ${rows}
+      </tbody>
+    </table>
+  </div>
+  <p><br data-mce-bogus="1"/></p>`;
+  }
```

Comment:
This should not belong inside the class. It is general enough that we can move it to a utility.

Also:
1. Add `contenteditable="false"` on `div` tag
2. Add `contenteditable="true"` on `table` tag